src/lib/features/project-status/project-status-controller.ts

```diff
@@ -0,0 +1,71 @@
+import type { Response } from 'express';
+import Controller from '../../routes/controller';
+import {
+    type IFlagResolver,
+    type IProjectParam,
+    type IUnleashConfig,
+    type IUnleashServices,
+    NONE,
+    serializeDates,
+} from '../../types';
+
+import { getStandardResponses } from '../../openapi/util/standard-responses';
+import type { OpenApiService } from '../../services';
+import type { IAuthRequest } from '../../routes/unleash-types';
+import {
+    createResponseSchema,
+    projectStatusSchema,
+    type ProjectStatusSchema,
+} from '../../openapi';
+import type { ProjectStatusService } from './project-status-service';
+
+export default class ProjectStatusController extends Controller {
+    private projectStatusService: ProjectStatusService;
+
+    private openApiService: OpenApiService;
+
+    private flagResolver: IFlagResolver;
+
+    constructor(config: IUnleashConfig, services: IUnleashServices) {
+        super(config);
+        this.projectStatusService = services.projectStatusService;
+        this.openApiService = services.openApiService;
+        this.flagResolver = config.flagResolver;
+
+        this.route({
+            method: 'get',
+            path: '/:projectId/status',
+            handler: this.getProjectStatus,
+            permission: NONE,
+            middleware: [
+                this.openApiService.validPath({
+                    tags: ['Projects'],
+                    operationId: 'getProjectStatus',
+                    summary: 'Get an overview of a project status.',
```

Comment:
Is this the endpoint for the whole project status view? 
Maybe something like this is enough:

```suggestion
                    summary: 'Get project status',
```